regression/cbmc-incr-smt2/bitvector-bitwise-operators/bitwise_ops.c

```diff
@@ -0,0 +1,24 @@
+int main()
+{
+  int a;
+  int b;
+
+  __CPROVER_assume(a != 0);
+
+  // This if going to be failing for values of `0000` and `1111`,
```

Comment:
:pick: The width of `int` is not 4 bits and is platform specific.
:pick: There is no guarantee that the values chosen for `a` and `b` will be those given in these comments. It just happens to be that the solver we used in our testing chose these values. These could be any non-zero value for `b` and any value for `a` where the bits set in `b` are not set in `a`.

Reply:
I know. These are just example values.

I will try to reword this in a way that makes this more explicit.

Reply:
Looks better. :+1: